### ISSUE TITLE
Pico cube wiki could do with filling out

### ISSUE THREAD

**Issue by mattcuk** [sbcshop/PICO-CUBE#2]
I'm raising the issue with the official [Pico Cube Wiki](https://learn.sb-components.co.uk/Pico-Cube) here, because I don't know where else to raise it :) 

Here's my feedback on the wiki;

- There is no pinout diagram in the Pinout section.
- There is no installation guide in the Installation section.
- It mentions running "demo.py", but this doesn't exist.. there are 2 demo files in the repo (this is a minor quibble)
- There really could do with a "Debugging your build" to help with any potential problems if the self-assembly doesn't fully work (strips of LEDs not working, individual LEDs not working, any other tips for diagnosing issues or sanity-checking the assembly)
- There's nothing listed in the Resources section.. not helpful!
- The sentence "learn the effects of LED lights with a different pattern of colours" is misleading.. these aren't RGB LEDs so you won't be able to create a "different pattern of colours".. it's one colour.. the colour you bought the kit in.
- The wiki can't be edited by anyone but SBC staff (no way for anyone to register), which doesn't help with someone like me coming along to help update it with the correct/better info.
- It could do with an overview/explanation of how kids can access individual LEDs, i.e. how the code actually works. There are virtually no comments in the demo code, and I think some diagrams in the wiki would really help kids conceptualise how the thing works.

Hope this can be addressed so that it helps anyone else with the product.

**Comment by misel228** [comment]
Yeah, I'm all for the wiki. While the SBC hardware is not that bad its community management and support on GH is lacking.

**Comment by sbcshop2** [comment]
Thank you for your valuable feedback, really appreciate it and we will update it accordingly.
